cmd/lakectl/cmd/local_clone.go

```diff
@@ -107,5 +110,6 @@ var localCloneCmd = &cobra.Command{
 func init() {
 	withGitIgnoreFlag(localCloneCmd)
 	withSyncFlags(localCloneCmd)
+	withForceFlag(localCloneCmd, "Force clone even on case-insensitive filesystem")
```

Comment:
We already have a "force" flag that does something differently.
Can we please rename it to something like `--ignore-case-insensitive` or something of the sorts?

Reply:
Changed to `--allow-case-insensitive`.